Makefile

```diff
@@ -3,18 +3,16 @@ IMAGE ?= poet
 BINARY := poet
 PROJECT := poet
 
+all: install build
+.PHONY: all
+
 BUF_VERSION := 1.8.0
 PROTOC_VERSION = 21.8
 PROTOC_GEN_GO_VERSION = v1.28
 PROTOC_GEN_GRPC_VERSION = v1.2
 PROTOC_GEN_GRPC_GATEWAY_VERSION = v1.16.0
 PROTOC_GEN_OPENAPIV2_VERSION = v2.12.0
 
-# `go install` will put binaries in $(GOBIN), avoiding
-# messing up with global environment.
-export GOBIN := $(PWD)/bin
-export PATH := $(GOBIN):$(PATH)
```

Comment:
I had to remove this or the build fails on non-linux systems: 
https://github.com/spacemeshos/poet/actions/runs/3291366658/jobs/5425450034

Reply:
I think it would just need a bit of tweaking to work in Windows and should work on Mac, shouldn't it? I can quickly prepare a version for windows for you, what do you think?

Reply:
The job failed on windows and mac - check the link 🙂.

What is the intention about installing the binaries to `./bin`? `go install pkg@version` ignores the `go.mod` file for building the tool and the dependencies are still cached and built in the same directory as the `poet` module. The only difference is that instead of putting the binary into `$(GOPATH)/bin` it's moved to `./bin`?

Reply:
The intention is to not install it globally. What if the developer uses different versions for each repository (which actually happens right now)? Because PATH is prepended with ./bin directory, all commands used in this Makefile will use these local binaries.

Reply:
The only tool I could find that is not in the same version everywhere is `protoc` which is not installed via `go install`.

Since `make install` is a prerequisite to `make build` and `make test` anyway. Why don't we just add it as a dependency to the `make` targets that need it? `go install` is a no-op anyway if instructed to install the same version as already on the system.

Reply:
How do you know if other people don't use other versions? IMO installing tools globally, and overwriting whatever is already installed, is wrong.

`protoc-gen-go` and `protoc-gen-grpc-gateway` are also of different versions on poet and api repos.

>go install is a no-op anyway if instructed to install the same version as already on the system.

On my system, it takes 13s to rerun `go install` of a single already installed tool.